lib/logstash/outputs/rabbitmq.rb

```diff
@@ -63,7 +63,7 @@ def multi_receive_encoded(events_and_data)
 
       def publish(event, message)
         raise ArgumentError, "No exchange set in HareInfo!!!" unless @hare_info.exchange
-        local_exchange.publish(message, :routing_key => event.sprintf(@key), :properties => symbolize(@message_properties.merge(:persistent => @persistent)))
+        local_exchange.publish(message, :routing_key => event.sprintf(@key), :properties => symbolize(@message_properties.merge(:persistent => @persistent), event))
```

Comment:
can we revert this change, I don't see a need to log the whole event

Reply:
I didn't understand what you meant.
I tried to evaluate the value in the {key: value}, so when it's a variable, like {"reply_to" => "%{myReplyTo}"}, the value in rabbit won't be "%{myReplyTo}" but the value of the variable myReplyTo.
I know there is an issue with the commit now (and the reason it didn't passed the tests), that it refers to the value always as a string, and if it's a boolean (like {"persistent" => true}) it will return  {"persistent" => "true"}.
I'll fix it. But is there any other things I'm missing?

Reply:
I didn't understand what you meant.
I tried to evaluate the value in the {key: value}, so when it's a variable, like {"reply_to" => "%{myReplyTo}"}, the value in rabbit won't be "%{myReplyTo}" but the value of the variable myReplyTo.
I know there is an issue with the commit now (and the reason it didn't passed the tests), that it refers to the value always as a string, and if it's a boolean (like {"persistent" => true}) it will return  {"persistent" => "true"}.
I'll fix it. But is there any other things I'm missing?